src/vm.rs

```diff
@@ -581,12 +584,162 @@ impl<T: GuestStagePageTable> Vm<T, VmStateFinalized> {
             ),
             SbiMessage::Base(base_func) => EcallAction::Continue(self.handle_base_msg(base_func)),
             SbiMessage::HartState(hsm_func) => self.handle_hart_state_msg(hsm_func),
-            SbiMessage::Tee(tee_func) => self.handle_tee_msg(tee_func, active_pages),
+            SbiMessage::Tee(tee_func) => self.handle_tee_msg(tee_func, active_cpu.active_pages()),
```

Comment:
unrelated change ?

Reply:
Ah, it's needed because we changed the type of input parameter to  active_cpu.